Sources/MapboxMaps/Location/Puck/PuckType.swift

```diff
@@ -32,6 +32,9 @@ public struct Puck2DConfiguration: Equatable {
     /// The color of the accuracy ring border.
     public var accuracyRingBorderColor: UIColor
 
+    /// Whether the puck rotates to track the bearing source.
+    public var puckBearingEnabled: Bool
+
```

Comment:
I think we can simplify this name. Since this is a property on Puck2DConfiguration, we don't need the word puck in the property name. Let's call it `bearingEnabled`.

Alternatively, this could be a property on `LocationOptions` instead of a property on `Puck2DConfiguration` and `Puck3DConfiguration`. @Chaoba thoughts on the pros/cons of this approach?

Reply:
Yeah, adding this property in `LocationOptions` is a better solution. Will try it.